lib/main.dart

```diff
@@ -1,65 +1,96 @@
 import 'package:flutter/material.dart';
 import 'package:get_it/get_it.dart';
-import 'package:wpi_campus/add_event.dart';
-import 'package:wpi_campus/ui/home.dart';
+import 'package:wpi_campus/ui/add_event.dart';
+import 'package:wpi_campus/model/home_event.dart';
+import 'package:wpi_campus/model/user_event.dart';
 import 'package:wpi_campus/model/repositories/data_repository.dart';
 import 'package:wpi_campus/model/repositories/firebase/firebase_data_repository.dart';
-import 'package:wpi_campus/ui/user_events.dart';
+import 'package:wpi_campus/ui/home_event_card.dart';
+import 'package:wpi_campus/ui/user_event_card.dart';
 
 final get = GetIt.instance;
 
+// todo consider using a material swatch instead of just primary color
+final _theme = ThemeData(primaryColor: Color.fromRGBO(172, 43, 55, 1));
+
```

Comment:
Should we make a theme.dart somewhere?

Reply:
IMO for a simple one-liner like this that ties into the app widget, this is fine

Reply:
Yee that makes sense. I think we should still have a theme.dart eventually when we get more complicated themes